src/providers/dropbox.js

```diff
@@ -0,0 +1,26 @@
+export default (options) => {
+  console.log('> Dropbox')
```

Comment:
You forgot a `console.log` :slightly_smiling_face:

Also, could you maybe use a named function with some JSDoc comments above? I have plans about extracting that information from these files, so we could auto-generate the documentation page for them!

See: https://github.com/nextauthjs/next-auth/blob/beta/src/providers/instagram.js